praw/models/reddit/subreddit.py

```diff
@@ -873,6 +877,120 @@ def submit(
 
         return self._reddit.post(API_PATH["submit"], data=data)
 
+    def submit_gallery(
+        self,
+        title,
+        images,
+        flair_id=None,
+        flair_text=None,
+        resubmit=True,
+        send_replies=True,
+        nsfw=False,
+        spoiler=False,
+        collection_id=None,
+        discussion_type=None,
+    ):
+        """Add an image gallery submission to the subreddit.
+
+        :param title: The title of the submission.
+        :param images: The images to post in dict with the following structure:
+            ``{"image_path": "path", "caption": "caption", "outbound_url": "url"}``,
+            only ``"image_path"`` is required.
+        :param collection_id: The UUID of a :class:`.Collection` to add the
+            newly-submitted post to.
+        :param flair_id: The flair template to select (default: None).
+        :param flair_text: If the template's ``flair_text_editable`` value is
+            True, this value will set a custom text (default: None).
+        :param resubmit: When False, an error will occur if the URL has already
+            been submitted (default: True).
+        :param send_replies: When True, messages will be sent to the submission
+            author when comments are made to the submission (default: True).
+        :param nsfw: Whether or not the submission should be marked NSFW
+            (default: False).
+        :param spoiler: Whether or not the submission should be marked as
+            a spoiler (default: False).
+        :param discussion_type: Set to ``CHAT`` to enable live discussion
```

Comment:
Does this parameter work for gallery type submissions?

Reply:
Yes. Reddit allows you to set all of these on the site.